bodhi/masher.py

```diff
@@ -589,10 +593,13 @@ def wait_for_sync(self):
             newsum = hashlib.sha1(masterrepomd.read()).hexdigest()
             if newsum == checksum:
                 log.info("master repomd.xml matches!")
-                notifications.publish(topic="mashtask.sync.done", msg=dict())
+                notifications.publish(topic="mashtask.sync.done", msg=dict(
+                    repo=self.id))
                 return
-            log.debug("master repomd.xml doesn't match! %s != %s",
-                      checksum, newsum)
+
+            # This seems like a pretty big deal..
+            log.error("master repomd.xml doesn't match! %s != %s for %r",
```

Comment:
Nope, it's expected that the repomd.xml won't match for up to 15min or so until the cron job syncs the repos to the master mirror.